cfgov/ask_cfpb/jinja2/ask-cfpb/answer-page.html

```diff
@@ -63,17 +63,6 @@ <h1>
                 </button>
             </div>
 
-            {% if page.related_resource and page.related_resource.trans_text(page.language) | striptags %}
-            <div class="block next-steps">
```

Comment:
It looks like you could remove the `.next-steps` class from the CSS in the ask less file at https://github.com/cfpb/consumerfinance.gov/blob/main/cfgov/unprocessed/css/on-demand/ask.less#L187

I'm not seeing another reference to next-steps in Ask (though it does appear in other apps, but they should have their own CSS for it).